public/javascript/main.js

```diff
@@ -376,5 +376,62 @@ $(function(){
 
           $(this).data('chart', chart);
       }
-  })
+  });
+
+  $(".js-download-with-term").on("click", function(e){
+    var term = $(e.target).data("term");
+    var house = $(e.target).data("house");
+    if (term && house) {
+        // inject query vars into URL before the target anchor
+        e.preventDefault();
+        location = $(e.target).attr('href')
+          .replace('#', "?" + $.param({term: term, house: house}) + "#");
+    }
+  });
+
+  if (typeof pageInitFunction === 'function') {
+      pageInitFunction();
+  }
+
 });
+
+// collapse_displayed_items:
+// If there are too many items being displayed in the list, hide
+// all but a few, making sure the target element (if there is one)
+// is included in those that are not hidden. Add a button for
+// revealing the hidden items.
+// Note this is used on the download page, where (currently) there
+// may be more than one such list (e.g., one for each legislature).
+
+function collapse_displayed_items(
+    $list_of_elements,   // list of elements that need to be collapsed
```

Comment:
`$list_of_elements` is slightly misleading here. I initially assumed it was a jQuery object containing an actual list element (like a `<ul>`). In reality, the "list" is the jQuery object itself (which isn't actually a list, in any meaningful sense of the word).

Perhaps rename to `$elements` to avoid confusion?

Or, `$items`, since you use the term `items` more frequently inside this function, and the fact that the variable name starts with a `$` already implies that it’s a jQuery object containing one or more DOM elements.